src/sync.js

```diff
@@ -528,13 +530,13 @@ SyncApi.prototype._syncFromCache = async function(savedSync) {
     };
 
     try {
-        await this._processSyncResponse(oldSyncToken, data, true);
+        await this._processSyncResponse(oldSyncEventData, data, true);
```

Comment:
why aren't we just using `syncEventData` here? it seems to be what the other call to `_processSyncResponse` does?

Reply:
No idea, we should be using `syncEventData`.